group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/TargetAssignmentBuilder.java

```diff
@@ -244,16 +245,18 @@ public TargetAssignmentResult build() throws PartitionAssignorException {
         });
 
         // Prepare the topic metadata.
-        Map<Uuid, AssignmentTopicMetadata> topics = new HashMap<>();
+        Map<Uuid, PartitionMetadata> topicMetadataMap = new HashMap<>();
```

Comment:
I was hoping that we could actually avoid this copy. Would it be possible?

Reply:
I don't think so